azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/CallLauncherActivity.kt

```diff
@@ -182,20 +228,58 @@ class CallLauncherActivity : AppCompatActivity() {
         }
     }
 
-    fun showAlert(message: String) {
-        runOnUiThread {
-            val builder = AlertDialog.Builder(this).apply {
-                setMessage(message)
-                setTitle("Alert")
-                setPositiveButton("OK") { _, _ ->
-                }
-            }
-            builder.show()
+    private fun saveState(outState: Bundle?) {
+        outState?.putBoolean(isTokenFunctionOptionSelected, callLauncherViewModel.isTokenFunctionOptionSelected)
+        outState?.putBoolean(isKotlinLauncherOptionSelected, callLauncherViewModel.isKotlinLauncher)
+    }
+
+    private val onSampleListener = object : MemoryMonitor.OnSampleListener {
```

Comment:
Can put this object in another file, doesn't directly pertain to CallLauncherActivity